Rain-Drops/style.css

```diff
@@ -0,0 +1,619 @@
+* {
```

Comment:
You use the   _font: bold ... Arial;_ 19th time.
What about making it a common style and just change a font-size for the text blocks?